sync/validation_queue.go

```diff
@@ -86,18 +86,18 @@ func (vq *blockQueue) handleBlocks(bjb fetchJob) {
 }
 
 func (vq *blockQueue) handleBlock(id types.Hash32, block *types.Block) {
-	vq.With().Info("start handling", block.ID(), block.MinerID())
+	vq.With().Info("start handling block", block.Fields()...)
 	if err := vq.fetchBlockDataForValidation(block); err != nil {
-		vq.Error("block fetching data failed", block.ID(), log.Err(err))
+		vq.With().Error("block data fetching failed", append(block.Fields(), log.Err(err))...)
```

Comment:
```suggestion
		vq.With().Error("fetching block data failed", append(block.Fields(), log.Err(err))...)
```

Reply:
funny, we've now gone through almost all possible positions for the word `fetching`... All accept `block data failed fetching`

Reply:
how about "fetching the fetching fetchers"? 😬

Reply:
totally unrelated but funny: https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo

Reply:
My English teacher once challenged us to punctuate [this one](https://en.wikipedia.org/wiki/James_while_John_had_had_had_had_had_had_had_had_had_had_had_a_better_effect_on_the_teacher) (I remember it with Tom and Bob, though).